### ISSUE TITLE
Bing images won't show with PHP 7.3.0

### ISSUE THREAD

**Issue by 19188** [yrccondor/mdx#22]
```
Arch# cat /var/log/httpd/error_log           

[Mon Dec 31 13:46:13.498056 2018] [proxy_fcgi:error] [pid 24838:tid 140699328489216] [client 172.68.142.89:40912] AH01071: Got error 'PHP message: PHP Warning:  preg_match(): The /e modifier is no longer supported, use preg_replace_callback instead in /srv/wordpress/wp-content/themes/mdx/index.php on line 10'

Arch# php-fpm --version
PHP 7.3.0 (fpm-fcgi) (built: Dec  8 2018 12:11:20)
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.0-dev, Copyright (c) 1998-2018 Zend Technologies
```

**Comment by yrccondor** [comment]
'Warning' means the code will be exceuted normally in most cases. The /e modifier was removed from PHP 7.0.0 and this function works normally in my tests (7.2.3) so it should works in your env. Please check your env or give me more information. The /e modifier will be remove in the next version anyway.

'Warning' 意味着大多数情况下代码可以按预期执行。/e 修饰符自 PHP 7.0.0 开始已经被移除，同时这个功能在我的测试中运行正常（7.2.3）。你可以检查一下你的环境配置或者告诉我更多信息。/e 修饰符会在下次更新中被移除。

**Comment by 19188** [comment]
> 'Warning' means the code will be exceuted normally in most cases. The /e modifier was removed from PHP 7.0.0 and this function works normally in my tests (7.2.3) so it should works in your env. Please check your env or give me more information. The /e modifier will be remove in the next version anyway.
> 
> 'Warning' 意味着大多数情况下代码可以按预期执行。/e 修饰符自 PHP 7.0.0 开始已经被移除，同时这个功能在我的测试中运行正常（7.2.3），所以这个功能在你的环境中也应该正常工作。你可以检查一下你的环境配置或者告诉我更多信息。/e 修饰符会在下次更新中被移除。

http://php.net/ChangeLog-7.php#7.3.0

> Removed support for the /e (PREG_REPLACE_EVAL) modifier.
I'm using 7.3.0.

**Comment by 19188** [comment]
Arch user. Having os upgraded yesterday,the code can't be executed.

**Comment by yrccondor** [comment]
Fine, I'll fix it s soon s possible.

**Comment by yrccondor** [comment]
Fixed in d241668. The new version is on the way.
